lib/service/service.go

```diff
@@ -392,8 +393,11 @@ type TeleportProcess struct {
 	// during in-process reloads.
 	id string
 
-	// log is a process-local log entry.
+	// log is a process-local logrus.Entry.
+	// Deprecated: use logger instead.
 	log logrus.FieldLogger
+	// logger is a process-local slog.Logger.
+	logger *slog.Logger
```

Comment:
I was tempted to suggest we ought to name these to distinguish them - but I think the Deprecation makes this alright.